sentry.conf.py

```diff
@@ -259,11 +259,18 @@
 SENTRY_WEB_HOST = '0.0.0.0'
 SENTRY_WEB_PORT = 9000
 SENTRY_WEB_OPTIONS = {
+    'http': '%s:%s' % (SENTRY_WEB_HOST, SENTRY_WEB_PORT),
+    'protocol': 'uwsgi',
+    # This is need to prevent https://git.io/fj7Lw
+    'uwsgi-socket': None,
+    'http-keepalive': True,
+    'add-header': 'Connection: Keep-Alive',
```

Comment:
You don't need to add this header because with http/1.1 keepalive is implied. If you _don't_ want keepalive, you need an explicit `Connection: close`.

Reply:
You are right, that said I was simply following the docs here: https://uwsgi-docs.readthedocs.io/en/latest/HTTP.html#http-keep-alive

Will remove and see if it still works (I think it would based on our discussions on Slack)